docker-images/get-info/src/perl5/Reporting/DB/Role.pm

```diff
@@ -0,0 +1,71 @@
+
+package Reporting::DB::Role;
```

Comment:
My expectation is that role will be a static table that we will be reading from quite often, but updating infrequently.